framework/ChiMesh/SweepUtilities/SweepBoundary/boundary_iso_homo.cc

```diff
@@ -0,0 +1,14 @@
+#include "sweep_boundaries.h"
+
+//###################################################################
+/**Returns a pointer to a heterogenous flux storage location.*/
```

Comment:
the filename has homo which I interpret as homogenous but the comment says heterogenous. I am confused.

Reply:
Oops. Great catch. Probably copy paste error